src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -645,10 +643,7 @@ def is_distributed(self) -> bool:
             from lightning_habana import HPUParallelStrategy
 
             distributed_strategy.append(HPUParallelStrategy)
-        is_distributed = isinstance(self.strategy, tuple(distributed_strategy))
-        if isinstance(self.accelerator, TPUAccelerator):
-            is_distributed |= self.strategy.is_distributed
```

Comment:
I am very confused by the previous logic here in the XLAStrategy. I couldn't tell what the intention was here.

Reply:
I wouldn't put much thought into it. Probably an artifact from ancient times